__tests__/addaccount-page.po.js

```diff
@@ -29,4 +30,42 @@ export default class AddAccountPage extends BasePage {
       .click(getByText(getTransValueByKey('INPUT_PUBLIC_ADDRESS_LABEL')))
       .click(Selector('button').withText(getTransValueByKey('ACTION_6')));
   }
+
+  async addPrivateKey() {
+    await this.navigateToPage();
+    await this.waitPageLoaded();
+    await t.click(getByText(getTransValueByKey('X_PRIVKEY2')));
+    this.waitForPage(PAGES.ADD_ACCOUNT_PRIVATE_KEY);
+
+    await t
+      .typeText(Selector('div[data-testid="selector"]').find('input'), FIXTURE_ETHEREUM)
+      .click(Selector('div[data-testid="selector"]').find('span').withText(FIXTURE_ETHEREUM))
+      .click(Selector('button').withText(getTransValueByKey('ACTION_6')))
+      .typeText(Selector(`div`).withText(getTransValueByKey('PRIVATE_KEY_PLACEHOLDER')).find('input'), FIXTURE_TEST_PRIVATE_KEY)
+
+    // Lose focus
+    await t.expect(Selector('button').withText(getTransValueByKey('ADD_LABEL_6_SHORT')).hasAttribute('disabled')).notOk('ready for testing', { timeout: FIXTURES_CONST.TIMEOUT });
+    await t.click(Selector('button').withText(getTransValueByKey('ADD_LABEL_6_SHORT')));
+  }
+
+  async addKeystoreFile() {
+    await this.navigateToPage();
+    await this.waitPageLoaded();
+    await t.click(getByText(getTransValueByKey('X_KEYSTORE2')));
+    this.waitForPage(PAGES.ADD_ACCOUNT_KEYSTORE);
```

Comment:
Prefer to make the expensive page navigation explicit in the test file.
It will be easier to identify bottle necks.